cli/di.container.ts

```diff
@@ -156,7 +157,7 @@ export default function configureContainer(args: any = {}) {
       if (fs.existsSync(join(contextPath, "src", "agent.ts"))) {
         // point to compiled javascript agent in output folder
         const tsConfigPath = join(contextPath, "tsconfig.json")
-        const { compilerOptions } = jsonc.parse(fs.readFileSync(tsConfigPath, 'utf8'))
+        const { compilerOptions } = parse(fs.readFileSync(tsConfigPath, 'utf8')) as any
```

Comment:
`jsonc.parse` is already returned as any. We could also probably directly cast this as an object too.